.travis.yml

```diff
@@ -1,23 +1,20 @@
+dist: xenial
 language: csharp
-dist: trusty
```

Comment:
I don't know anything about Travis so I've got a couple of questions about this to help me learn :)

1. How did you decide on the dist and why did you change it in this PR?
2. Why is the language specified as csharp and not fsharp?